src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -255,6 +255,11 @@ void DlgPrefInterface::slotUpdate() {
 
     int inhibitsettings = static_cast<int>(m_mixxx->getInhibitScreensaver());
     comboBoxScreensaver->setCurrentIndex(comboBoxScreensaver->findData(inhibitsettings));
+
+    bool effectLoadMetaknobBehavior = m_pConfig->getValue(
+            ConfigKey("[Effects]", "EffectLoadMetaknobBehavior"), true);
```

Comment:
"behavior" is not a good naming for a boolean value.

When switching effects we can only preserve one of the following knob positions without inconsistencies:
- effect parameters
- metaknob
- superknob

You choose one these. The positions of the chosen one and all following in downwards direction are kept/preserved, while the corresponding parameters of all others in upwards direction are adjusted/adopted accordingly. Currently I don't see any more possible choices. The ordering with the implications might even give hints how a good user interface for this selection might look like.

Just brainstorming in the morning to help you find a good name.